buildSrc/src/main/kotlin/ort-kotlin-conventions.gradle.kts

```diff
@@ -112,7 +112,6 @@ detekt {
 java {
     toolchain {
         languageVersion = JavaLanguageVersion.of(javaLanguageVersion)
-        vendor = JvmVendorSpec.ADOPTIUM
```

Comment:
Does this commit fix the issue, which has been discussed in slack today regarding "Cannot find a Java installation on your machine ..." ? If so, should the commit message mention this?

Reply:
It should fix that issue, yes. But as others have reported this before, see e.g. https://github.com/oss-review-toolkit/ort/issues/9293, I've rather mentioned that issue in the commit message.